### DISCUSSION TITLE
About "first_saccade_only(bool)"

### DISCUSSION THREAD

**Discussion by mountains-high** [neuromorphs/tonic#115] [Q&A] [answered]
Good evening,

Thank you for the package!
I'd like to ask:
1.	How to set if we would like to use 2nd saccade only. The same for if we would like to use only 3rd saccade for all DVS datasets. 
2.	How to set if we’d like to use 1st saccade with only “ON” events
The same for “OFF” events of 1st saccade? I wanted to use only “ON” events of the 3rd saccade of the NMNIST dataset. 

Thank you!
Assel

**Comment by biphasic** [comment, marked as answer]
Hey Assel,
1. if you want to use 2nd or 3rd saccade in the nmnist dataset, I suggest to load the full recording and then select the events you need along the time axis. You can see how the first saccade is selected in the dataset class: https://github.com/neuromorphs/tonic/blob/develop/tonic/datasets/nmnist.py Currently only NMNIST and NCALTECH101 datasets were recorded using saccades. I suggest to have a look into the respective papers to understand how they were recorded
2. Same as for 1., you can just load all the events and then select the ones you need before feeding them to your algorithm.

**Reply by mountains-high** [reply]
Thank you very much @biphasic
I understood and will try regarding ans. 1, but could you a bit illustrate q2. please? 

From the paper [Is Neuromorphic MNIST neuromorphic? Analyzing discriminative power of neuromorphic datasets in the time domain, Laxmi et al. ] ```In our experiments, we consider ON events in the first saccade( 0-105ms) only.``` So it means that first saccade stands for ON events and 2nd for OFF events? If I'm wrong, could you correct me, please? 
Thank you very much!
